app/src/main/java/io/appium/uiautomator2/utils/Attribute.java

```diff
@@ -41,6 +41,7 @@
     SELECTION_END(new String[]{"selection-end"}),
     SELECTED(new String[]{"selected"}),
     TEXT(new String[]{"text", "name"}),
+    HINT(new String[]{"text", "hint"}),
```

Comment:
Please remove "text" from this array as it might create a serious confusion in the downstream code if other items have the same alias

Reply:
yea, that was my wrong